macros/contexts/contextSignificantFigures.pl

```diff
@@ -162,6 +165,11 @@ sub TeX {
 	return "{$tex}";
 }
 
+sub expFormat {
+	my $self = shift;
+	return $self->format('E', $self->value, $self->N);
+}
```

Comment:
Note that the `$self->format('E')` is equivalent to `$self->format('E', $self->value, $self->N)`, so this new method is basically redundant, as `$self->format('E')` is no harder to do that `$self->expFormat`.  I'm not sure there is a need for this method, especially since your only use is in the test files.

Reply:
Actually $n is not the place to round (it turns into that at the end as `$n-1`), however I'm not getting the same results between `$self->format('E')` and `$self->format('E', $self->value, $self-N)` I think because of the line I sited above.